website/docs/reference/change-requests.mdx

```diff
@@ -10,171 +10,104 @@ import VideoContent from '@site/src/components/VideoContent.jsx';
+Once a scheduled change is applied, both the person who scheduled it and the change request author receive a notification.
 
-#### Edge cases: what happens when ...?
+Scheduling changes using change requests gives you a centralized view of changes across multiple flags and strategies, making it easy to reschedule or reject them. However, [potential conflicts](#scheduling-errors) can cause the scheduling to fail.
 
-If the user who scheduled a change request is deleted from the Unleash users list before the scheduled time, the changes will **not** be applied. Instead, the schedule will be put into a special **suspended state**. A change request with suspended schedule will not be applied at its scheduled time. A user with the required permission can reschedule, apply, or reject the change request. Any of these actions will put the change request back into the regular flow.
+Alternatively, you can use constraints or segments with the `DATE_AFTER` [operator](/reference/activation-strategies#constraint-operators). This approach avoids conflicts, ensures SDKs are aware of changes in advance, and allows them to apply the changes even if they temporarily fail to connect to Unleash.
 
-If a change request has been scheduled and change requests are then disabled for the project and environment, the change request **will still be applied** according to schedule. To prevent this, you can reject the scheduled change request.
+### Scheduling errors
 
+Unleash suspends a scheduled change request if:
+- The change request includes updates to a flag that has been archived or a strategy that has been deleted. 
+- The change request includes a strategy, segment, or variant that has been updated.
+- The user who scheduled a change request is deleted from the users list before the scheduled time.
 
-#### Different ways to schedule changes
+You must manually review suspended change requests to reschedule, apply, or reject them.
 
-Unleash currently offers two distinct ways to schedule changes. Each method has its own pros and cons, and you can also combine the methods for maximum flexibility.
+For any suspended or failed change requests, Unleash sends out email notifications to the change request author and to the person who scheduled the change request. Additionally, Unleash displays warnings for flag or strategy deletions that conflict with existing scheduled change requests.
 
-The first method is through scheduled change requests, as we have explained in the preceding sections. Scheduled change requests make it easy to see all the changes across multiple flags and strategies in one view and makes it easy to reschedule or reject them. However, because scheduled changes rely on flags and strategy configurations, conflicts can arise causing the schedule to fail.
+If Unleash fails to apply a scheduled change request, the change request remains in the scheduled state. You can either reschedule and attempt to apply it again or reject it.
```

Comment:
Is there any other reason it would 'fail' besides the above errors that cause the suspended state? Is there a reason to differentiate here?

Reply:
Good question! In an ideal world: no. Any other change should go through. But there may be unforeseen circumstances that make it not work (or there may be edge cases that we haven't foreseen). So it's more of a fallback that you hopefully shouldn't ever see than anything else.